packages/chatbot-server-mongodb-public/src/tools/fetchPage.ts

```diff
@@ -109,3 +81,57 @@ export function makeFetchPageTool(
     ),
   });
 }
+
+async function getPageContent(
```

Comment:
note that for here it doesnt really matter if the params are an object or the current set b/c this is an internal method. so doesnt really change